docs/UserGuide.md

```diff
@@ -382,7 +382,8 @@ The simplest way to analyze multiple repos in one go is to use the `--repos` par
 
 Alternatively, you can use csv config files to customize the analysis as before while specifying multiple repos to analyze.
 * `repo-config.csv`: Add additional rows for the extra repos ([example](repo-config.csv))
-* `author-config.csv`: Add one row for each author in each repo you want to analyze
+* `author-config.csv`: Add one row for each author in each repo you want to analyze ([example](author-config.csv))
+* `group-config.csv`: Add one row for each repo which you want to group ([example](group-config.csv))
```

Comment:
Need to be more specific, so that the users understand what are the values that group-config brings.

In fact, all the other descriptions should also be updated as well to be kept up to date. E.g. give a better general description or list out some of the features.